### DISCUSSION TITLE
Failed to initialize snail

### DISCUSSION THREAD

**Discussion by mid-at-coding** [minexew/Shrine#57] [Q&A]
![image](https://user-images.githubusercontent.com/64973075/108412757-2f0ad800-71f8-11eb-8c83-1f81e8238e35.png)
how do i fix this (host is linux x86_64 xubuntu\)

**Comment by minexew** [comment]
Tell me more -- are you actually trying to set up networking over virtual serial, or would you like to use an emulated Ethernet card?

What is the exact command that you are using to run the VM? Etc.
